test/action_controller/serialization_scope_name_test.rb

```diff
@@ -4,8 +4,10 @@
 class DefaultScopeNameTest < ActionController::TestCase
   class UserSerializer < ActiveModel::Serializer
     attributes :admin?
-    def admin?
-      current_user.admin
+    ActiveModelSerializers.silence_warnings do
+      def admin?
+        current_user.admin
+      end
```

Comment:
I couldn't figure out how to get around this 'method redefined?' warning. I think Rails silences these kind of warnings as well.